### ISSUE TITLE
border-radius gets incorrectly abbreviated

### ISSUE THREAD

**Issue by ForNeVeR** [jakubpawlowicz/clean-css#765]
Consider this CSS:

```css
.button {
    border-top-left-radius: 100px 24px;
    border-top-right-radius: 100px 24px;
    border-bottom-left-radius: 100px 24px;
    border-bottom-right-radius: 100px 24px;
}
```

After being processed with clean-css using command `cleancss -o test.min.css test.css` it becomes:

```css
.button{border-radius:100px}
```

I don't think that's correct (because `24px` was lost). Maybe it should become something like `border-radius:100px/24px`, but definitely not this.

**Comment by Newlifer** [comment]
I second this.

**Comment by jakubpawlowicz** [comment]
Ooops, this is clearly wrong. Fix coming!

**Comment by jakubpawlowicz** [comment]
Fixed in 3.4.14.
